packages/shared/src/eventBus/appEventBus.ts

```
@@ -326,6 +326,15 @@ class AppEventBus extends CrossEventEmitter {
       defaultLogger.app.eventBus.emitToSelf({
         eventName: type,
       });
+      try {
+        // @ts-ignore
+        if (payload?.$$isRemoteEvent) {
+          // @ts-ignore
+          delete payload.$$isRemoteEvent;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)